src/user/models/user/dto/linked-user.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { IsBoolean, IsEnum, IsNotEmpty, IsString, Matches } from 'class-validator';
+import { Blockchain } from 'src/ain/services/crypto.service';
+import { GetConfig } from 'src/config/config';
+
+export class LinkedUserDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  @Matches(GetConfig().addressFormat)
+  address: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsEnum(Blockchain)
+  blockchain: Blockchain;
+
+  @ApiPropertyOptional()
+  @IsBoolean()
+  isSwitchable: boolean;
+}
```

Comment:
Would it make sense to split this in a in- and out-DTO? Might be disturbing to have the `isSwitchable` in the in-DTO.

Reply:
done